corgi/tasks/managed_services.py

```diff
@@ -0,0 +1,109 @@
+from celery_singleton import Singleton
+from django.db import transaction
+from django.utils import timezone
+
+from config.celery import app
+from corgi.collectors.app_interface import AppInterface
+from corgi.collectors.syft import Syft
+from corgi.core.models import (
+    Component,
+    ComponentNode,
+    ProductComponentRelation,
+    ProductStream,
+    SoftwareBuild,
+)
+from corgi.tasks.common import RETRY_KWARGS, RETRYABLE_ERRORS
+from corgi.tasks.sca import save_component
+
+
+@app.task(base=Singleton, autoretry_for=RETRYABLE_ERRORS, retry_kwargs=RETRY_KWARGS)
+def refresh_service_manifests():
+    services = ProductStream.objects.filter(meta_attr__managed_service_components__isnull=False)
+    service_metadata = AppInterface.fetch_service_metadata(services)
+
+    for service, components in service_metadata.items():
+        manifest_service.delay(str(service.pk), components)
+
+
+@app.task(base=Singleton, autoretry_for=RETRYABLE_ERRORS, retry_kwargs=RETRY_KWARGS)
+def manifest_service(product_stream_id: str, service_components: list):
```

Comment:
Let's run this task in the CPU queue since it needs to download and scan a potentially large Quay image / Git repo, and since there are many copies of it running at once:
```suggestion
def cpu_manifest_service(product_stream_id: str, service_components: list) -> None:
```